plugins/profile.py

```diff
@@ -91,6 +91,20 @@ async def setprofile(msg: Message) -> None:
         await msg.conn.main_room.send_rankhtmlbox("%", message)
 
 
+@command_wrapper(
+    aliases=("clearprofilo",), helpstr="Rimuovi la frase personalizzata dal profilo."
+)
```

Comment:
May I suggest adding `resetprofile`/`resetprofilo` as a secondary alias? I think it's less understandable than `clearprofile` but people might still write it that way. I don't really foresee any other useful aliases, as the usual prefixes we use for this kind of commands (`del-`, `rm-`) aren't meaningful in this context.
```suggestion
@command_wrapper(
    aliases=("clearprofilo", "resetprofile", "resetprofilo"),
    helpstr="Rimuovi la frase personalizzata dal profilo.",
)
```